### ISSUE TITLE
I'm having a problem with Join

### ISSUE THREAD

**Issue by curlyfro** [rosenbjerg/FFMpegCore#80]
i'm trying this in a test project.  this is your input.mp4 file.

```
            FFMpeg.Join(@"..\..\..\joined_video.mp4",
                @"..\..\..\videos\output\input.mp4"
            );
```

this is error I'm getting

FFMpegCore.Exceptions.FFMpegException : Invalid output file. File extension should be '.mpegts' required.

**Comment by pleshw** [comment]
Same error with following code:

```csharp
string output = "data/resultJoinVideo.mp4";
string[] input = "assets/test.mp4 data/resultJoinImage.mp4".Split(' ');
FFMpeg.Join(output, input);
```
_**FFMpegCore.dll: 'Invalid output file. File extension should be '.mpegts' required.'**_

**Comment by rosenbjerg** [comment]
Which version of the library and ffmpeg are you using?

**Comment by rosenbjerg** [comment]
I have just released version 2.2.2 which should fix this issue.
The issue was caused by ffmpeg builds naming the mpegts container extension differently (mpegts vs ts)
